src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.product {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $card-color;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: $main-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    color: $secondary-color;
+  }
+
+  &__review {
+    margin-bottom: 24px;
+    display: flex;
+
+    &-count {
+      margin-left: auto;
+      align-self: center;
+      text-align: center;
+      color: $main-color;
+    }
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-title {
+      color: $secondary-color;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-amount {
+      color: $main-color;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__order {
+    height: 40px;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid transparent;
```

Comment:
You already so the transparent border is not needed as box-sizing: border-box; rule already includes the width of the border in the overall width of the element